testsuite/dockerfiles/controller-dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensuse/leap:15.4
+FROM opensuse/leap:15.5
 RUN zypper ref -f && \ 
     zypper -n ar --no-gpgcheck http://download.opensuse.org/repositories/systemsmanagement:/sumaform:/tools/openSUSE_Leap_15.4/ tools && \
```

Comment:
```suggestion
    zypper -n ar --no-gpgcheck http://download.opensuse.org/repositories/systemsmanagement:/sumaform:/tools/openSUSE_Leap_15.5/ tools && \
```

Reply:
@jordimassaguerpla I also have a doubt on the list of installed packages.
`postgresql-devel` -> this should still be version 14 because we didn't bump the postgresql version.